inference/core/entities/requests/owlv2.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Optional, Union
+
+from pydantic import BaseModel, Field, validator
+
+from inference.core.entities.common import ApiKey
+from inference.core.entities.requests.inference import (
+    BaseRequest,
+    InferenceRequestImage,
+)
+from inference.core.env import OWLV2_VERSION_ID
+
+
+class TrainBox(BaseModel):
+    x: int
+    y: int
+    w: int
+    h: int
+    cls: str
+
+
+class TrainingImage(BaseModel):
+    boxes: List[TrainBox]
+    image: InferenceRequestImage
+
+
+class OwlV2InferenceRequest(BaseRequest):
+    """Request for gaze detection inference.
```

Comment:
Please use `Field(...)` with descriptions and defaults placed there, that is to push the info into OpenAPI docs